pkg/scheduler/predicates/ha.go

```diff
@@ -128,18 +129,21 @@ func (h *ha) Filter(instanceName string, pod *apiv1.Pod, nodes []apiv1.Node) ([]
 		// replicas less than 3 cannot achieve high availability
 		if replicas < 3 {
 			minNodeNames = append(minNodeNames, nodeName)
+			glog.Infof("replicas is %d, add node %s to minNodeNames", replicas, nodeName)
```

Comment:
Logs print here are unnecessary, at line 109 we know the replicas of the component, if the replicas is smaller than 3, all nodes will be added into `minNodeNames`. 

How about making it simpler, we can skip the for loop if the replicas is smaller than 3 and print an informative log?

Reply:
How to skip the `for` loop? we must range the `nodeMap` anyway

Reply:
hmm, you're right, we need to get node name